### PR TITLE
[WFCORE-4974] Bump the org.wildfly.core.testrunner.Server start/stop …

### PR DESCRIPTION
**Pull request by bstansberry**
…timeouts

https://issues.redhat.com/browse/WFCORE-4974

### CODE REVIEW

**Comment by bstansberry** [comment]
@jamezp @jmesnil I pulled these new numbers out of thin air. If there are equivalent defaults in the ARQ containers, aligning would make sense just as a 'one less variation to keep in mind' kind of thing. I think a min of 60s for a start timeout is the right number though, not less.

**Comment by bstansberry** [comment]
I killed CI on this because we need the resources elsewhere today. We can trigger a 'retest' later.

File: testsuite/test-runner/src/main/java/org/wildfly/core/testrunner/Server.java [hunk 1]
```diff
@@ -66,8 +66,8 @@
     private final String managementProtocol = System.getProperty("management.protocol", "remote+http");
 
     // timeouts
-    private final int startupTimeout = Integer.getInteger("server.startup.timeout", 30);
-    private final int stopTimeout = Integer.getInteger("server.stop.timeout", 10);
+    private final int startupTimeout = Integer.getInteger("server.startup.timeout", 60);
```
Comment (jmesnil) [inline testsuite/test-runner/src/main/java/org/wildfly/core/testrunner/Server.java:69]: I'm fine with doubling these values.

I also wonder whether we could introduce a mechanism similar to `org.jboss.as.test.shared.TimeoutUtil` to influence the timeouts with the `ts.timeout.factor` system property?

Comment (bstansberry) [reply testsuite/test-runner/src/main/java/org/wildfly/core/testrunner/Server.java:69]: Makes sense. We'd need to port TimeoutUtil though as I don't think we want to add the dep on testsuite-shared.  That's no big deal though.

**Comment by bstansberry** [comment]
retest this please

File: testsuite/test-runner/src/main/java/org/wildfly/core/testrunner/Server.java (hunk 1 above)
Comment (bstansberry) [reply testsuite/test-runner/src/main/java/org/wildfly/core/testrunner/Server.java]: I updated the commit with the relevant code added directly to the Server class.

Comment (jamezp) [reply testsuite/test-runner/src/main/java/org/wildfly/core/testrunner/Server.java]: I'm pretty sure the `TimeoutUtil` is already in core right? https://github.com/wildfly/wildfly-core/blob/master/testsuite/shared/src/main/java/org/jboss/as/test/shared/TimeoutUtil.java

Comment (jmesnil) [reply testsuite/test-runner/src/main/java/org/wildfly/core/testrunner/Server.java]: That's correct, it's in core but for classes shared by the test suite. afaict, the testrunner does not depend on these shared classes.

**Pull request merged**